### PR TITLE
mgr/dashboard: Add cypress env for login credentials

### PR DESCRIPTION
**Pull request by Sarthak0702**
Fixes: https://tracker.ceph.com/issues/55270
Signed-off-by: Sarthak0702 <sarthak.dev.0702@gmail.com>

## Checklist
- Tracker (select at least one)
  - [x] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [x] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [ ] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Comment by Sarthak0702** [comment]
jenkins test api

**Review by epuertat — APPROVED** [review]

**Review by pereman2 — APPROVED** [review]

**Comment by avanthakkar** [comment]
jenkins test api

**Comment by Sarthak0702** [comment]
jenkins test api

**Comment by ljflores** [comment]
@Sarthak0702 could http://pulpito.front.sepia.ceph.com/yuriw-2022-04-13_20:41:44-rados-wip-yuri2-testing-2022-04-13-0703-distro-default-smithi/6790772/ be related to this PR?

/a/yuriw-2022-04-13_20:41:44-rados-wip-yuri2-testing-2022-04-13-0703-distro-default-smithi/6790772/teuthology.log
```
2022-04-13T21:07:50.959 INFO:tasks.workunit.client.0.smithi022.stdout:  1) Hosts page
2022-04-13T21:07:50.959 INFO:tasks.workunit.client.0.smithi022.stdout:       when Orchestrator is available
2022-04-13T21:07:50.959 INFO:tasks.workunit.client.0.smithi022.stdout:         "before each" hook for "should not add an exsiting host":
2022-04-13T21:07:50.960 INFO:tasks.workunit.client.0.smithi022.stdout:     CypressError: `cy.request()` failed on:
2022-04-13T21:07:50.960 INFO:tasks.workunit.client.0.smithi022.stdout:
2022-04-13T21:07:50.960 INFO:tasks.workunit.client.0.smithi022.stdout:https://172.21.15.22:8443/api/auth
2022-04-13T21:07:50.960 INFO:tasks.workunit.client.0.smithi022.stdout:
2022-04-13T21:07:50.960 INFO:tasks.workunit.client.0.smithi022.stdout:The response we received from your web server was:
2022-04-13T21:07:50.961 INFO:tasks.workunit.client.0.smithi022.stdout:
2022-04-13T21:07:50.961 INFO:tasks.workunit.client.0.smithi022.stdout:  > 500: Internal Server Error
2022-04-13T21:07:50.961 INFO:tasks.workunit.client.0.smithi022.stdout:
2022-04-13T21:07:50.961 INFO:tasks.workunit.client.0.smithi022.stdout:This was considered a failure because the status code was not `2xx` or `3xx`.
2022-04-13T21:07:50.961 INFO:tasks.workunit.client.0.smithi022.stdout:
2022-04-13T21:07:50.962 INFO:tasks.workunit.client.0.smithi022.stdout:If you do not want status codes to cause failures pass the option: `failOnStatusCode: false`
2022-04-13T21:07:50.962 INFO:tasks.workunit.client.0.smithi022.stdout:
```

/a/yuriw-2022-04-13_20:41:44-rados-wip-yuri2-testing-2022-04-13-0703-distro-default-smithi/6790772/remote/smithi022/log/884235d0-bb6c-11ec-8c38-001a4aab830c/ceph-mgr.a.log.gz
```
2022-04-13T21:07:48.827+0000 7fa9b6e5d700 20 mgr get_config  key: mgr/dashboard/AUDIT_API_ENABLED
2022-04-13T21:07:48.827+0000 7fa9b6e5d700 10 mgr get_typed_config  AUDIT_API_ENABLED not found
2022-04-13T21:07:48.828+0000 7fa9b6e5d700  0 [dashboard ERROR exception] Internal Server Error
Traceback (most recent call last):
  File "/usr/share/ceph/mgr/dashboard/services/exception.py", line 47, in dashboard_exception_handler
    return handler(*args, **kwargs)
  File "/lib/python3.6/site-packages/cherrypy/_cpdispatch.py", line 54, in __call__
    return self.callable(*self.args, **self.kwargs)
  File "/usr/share/ceph/mgr/dashboard/controllers/_base_controller.py", line 263, in inner
    ret = func(*args, **kwargs)
  File "/usr/share/ceph/mgr/dashboard/controllers/_rest_controller.py", line 191, in wrapper
    return func(*vpath, **params)
TypeError: create() missing 1 required positional argument: 'password'
```

File: src/pybind/mgr/dashboard/frontend/cypress.json [hunk 1]
```diff
@@ -18,5 +18,9 @@
       "mochaFile": "cypress/reports/results-[hash].xml"
     }
   },
-  "retries": 1
+  "retries": 1,
+  "env": {
+    "LOGIN_USER": "admin",
+    "LOGIN_PASSWORD": "admin"
```
Comment (ljflores) [inline src/pybind/mgr/dashboard/frontend/cypress.json:24]: If this PR is indeed the cause, my thought is that "LOGIN_PASSWORD" should be "LOGIN_PWD" to match what's written below.

Comment (Sarthak0702) [reply src/pybind/mgr/dashboard/frontend/cypress.json:24]: @ljflores Yup, that's correct looks like it somehow missed our radar!

**Pull request merged**